recipe/meta.yaml

```diff
@@ -30,20 +30,20 @@ outputs:
         - setuptools
         - wheel
         - packaging
-        - param 1.9.0
+        - param >=1.9.0,<3.0
         - pyct 0.5.0
-        - bokeh 2.4.3
-        - nodejs 16.13.1
-        - pyviz_comms 0.7.2
-        - cartopy 0.21.1
+        - bokeh >=3.3.0,<3.4
+        - nodejs 18.16
       run:
         - python
-        - bokeh >=2.4.0,<2.5.0
+        - bokeh >=3.2.0,<3.4.0
         - cartopy >=0.21.1
-        - holoviews >=1.14.2
-        - numpy
-        - param >=1.9.2,<2.0
-        - scipy
+        - holoviews >=1.16.0
+        - numpy >=1.0
+        - param >=1.9.0,<3.0
+        - panel >=1.0.0
+        - pyproj
+        - packaging
```

Comment:
Missing `xyzservices`, see https://github.com/holoviz/geoviews/blob/6c889dd42501d11abea2377ce79990bf69806b99/setup.py#L106

Reply:
Ush good catch I was fooled by the conda-forge recipe.

Reply:
Added it to the recipe.